civihr_default_theme/scss/base/_task-assignments-modal-temp.scss

```diff
@@ -63,6 +63,11 @@ $white: #FFFFFF;
       text-align: center;
     }
 
+    .no-file {
+      color: red;
```

Comment:
We should use existing color variables here.
Example: bootstrap danger color for red